pages/common/upx.md

```diff
@@ -0,0 +1,15 @@
+# upx
+
+> Compress or expand executables.
```

Comment:
Should we change it to `Compress or decompress executables` to maintain consistency with the later description ?

Reply:
Not sure how I missed that typo. 🤔 But yes, probably would be good. 👍

Reply:
Made this change in 2122b5b451e03afdecf9ac90aec4d7f6f170738b, should be fine now :+1: